utils/lib/nr-graphql-helpers.ts

```diff
@@ -175,6 +175,39 @@ export const getCategoryTermsFromKeywords = async (
   return categoryKeywords.length > 0 ? categoryKeywords : undefined;
 };
 
+type CoreDataSourceSearchResults = {
```

Comment:
@MichelLosier and I were talking about this while working on dashboards but it might be nice in the future if we could use codegen in order to generate these types from our queries (similar to how we do it in the marketplace?)